galley/cmd/galley/cmd/root.go

```diff
@@ -156,6 +156,8 @@ func GetRootCmd(args []string) *cobra.Command {
 		serverArgs.SinkAuthMode, "Name of authentication plugin to use for connection to sink server.")
 	rootCmd.PersistentFlags().StringSliceVar(&serverArgs.SinkMeta, "sinkMeta",
 		serverArgs.SinkMeta, "Comma-separated list of key=values to attach as metadata to outgoing sink connections. Ex: 'key=value,key2=value2'")
+	rootCmd.PersistentFlags().BoolVar(&serverArgs.UpgradeAuthenticationPolicyToV2, "upgradeAuthenticationPolicyToV2", serverArgs.UpgradeAuthenticationPolicyToV2,
```

Comment:
Why is this a flag? For development purposes? Then we should probably make it a temporary environment flag.

Reply:
Yes it's for development purpose. Also we don't want to accidentally mess up with the current flow. If it's okay to use a environment variable, I'm happy to change it.

Reply:
Okay, so let me drop this environment variable completely.